src/NumberTheory/GaloisGrp/GaloisGrp.jl

```diff
@@ -2371,7 +2371,7 @@ end
 """
     galois_quotient(C::GaloisCtx, Q::PermGroup)
 
-Finds all(?) subfields of the splitting field s.th. the galois
+Find all(?) subfields of the splitting field s.th. the galois
```

Comment:
Non-blocking comment:
The question mark in brackets after the word `all` is not helpfull for the reader. Either be sure what you return or specify more clearly, what the doubt is.

Reply:
Agreed, and "s.th." should be spelled out, and "Q" in the next line be changed to either "the rationals" or `$\mathbb{Q}$`. Alas that's getting quite out of the scope of this PR. I also don't know what the `(?)` means -- I am guessing we need to ask @fieker about that.